### PR TITLE
Explain edge directionality in graph docs

### LINKED ISSUE #1049: Explain edge directionality
We need to document the way in which we conceptually treat each edge as having a separate "forward" and "backward" component. This should likely go in a docstring inside the Graph module and then be referenced elsewhere.

Thanks to @anthrocypher for surfacing this here: https://github.com/sourcecred/sourcecred/pull/1043#discussion_r247744231

### PR DESCRIPTION
**Pull request by decentralion**
Fixes #1049

Test plan: Review by @anthrocypher

### CODE REVIEW

File: src/core/graph.js [hunk 1]
```diff
@@ -62,12 +62,38 @@ import * as NullUtil from "../util/null";
...
+ * const me = NodeAddress.fromParts(["USER", "decentralion"]);
+ * const edgeParts = ["AUTHORS", "USER", "decentralion", "PULL_REQUEST", "1"];
+ * const authorsAddress = EdgeAddress.fromParts(edgeParts);
+ * const authorsEdge = {src: me, dst: pr, address: authorsAddress};
  *
- * Creating a graph is as simple as invoking the constructor and adding nodes and edges:
+ * Each edge is directed, in the sense that it goes from the `src` node to the
+ * `dst` node. For example, the "AUTHORS" edge in the example above goes from
+ * the authoring user (@decentralion) to the authored pull request (PR #1).
+ * When doing cred analysis, we actually treat each edge as a pair of edgtes
```
Comment (BrianLitwin) [inline src/core/graph.js]: spelling: "edgtes"

File: src/core/graph.js [hunk 2]
```diff
@@ -62,12 +62,38 @@ import * as NullUtil from "../util/null";
...
+ * edges: a "forward" edge that brings cred from the source to the destination,
+ * and a "backward" edge that brings cred from the destination back to the source.
+ * In the example above, the forward edge gives brings cred from a user to a
```
Comment (BrianLitwin) [inline src/core/graph.js]: semi-unclear: "the forward edge gives brings cred from a user'

File: src/core/graph.js [hunk 3]
```diff
@@ -62,12 +62,38 @@ import * as NullUtil from "../util/null";
...
+ * post they authored, and the backwards edge brings cred from a post to the
```
Comment (BrianLitwin) [inline src/core/graph.js]: Is "brings" cred more clear/accurate than "transfers" cred?

**Comment by decentralion** [comment]
I'm closing this per @wchargin's suggestion. edge directionality is now discussed here: https://github.com/sourcecred/research/blob/master/algorithm.md#edge-bidirectionality

**Pull request closed without merging**